integration/network/disable_net/net_none.bats

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../../lib/common.bash"
+
+setup () {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "Disable net" {
+	IMAGE="busybox"
+	PAYLOAD="true"
+
+	extract_kata_env
+
+	# Run a container with network
+	run docker run -d --rm --runtime=${RUNTIME} ${IMAGE} ${PAYLOAD}
+	[ "$status" -eq 0 ]
+
+	# Get the name of the network name at the configuration.toml
+	NETWORK_NAME=$(grep -E "internetworking_model=" /usr/share/defaults/kata-containers/configuration.toml | head -1 | cut -d '"' -f2)
+
+	# Disable the network
+	sudo sed -i 's/#disable_new_netns = true/disable_new_netns = true/g' ${RUNTIME_CONFIG_PATH}
+	sudo sed -i 's/internetworking_model=".*"/internetworking_model="none"/g' ${RUNTIME_CONFIG_PATH}
+
+	# Run a container without network
+	run docker run -d --rm --runtime=${RUNTIME} ${IMAGE} ${PAYLOAD}
```

Comment:
From the commit - did you intend to run this command with '--net=none' as well? Maybe it is implicit somewhere/somehow?